src/python/pants/option/config_test.py

```diff
@@ -73,21 +73,21 @@ def test_getstring(self):
 that.""",
       self.config.get('b', 'disclaimer'))
 
-    self._check_defaults(self.config.get, '')
-    self._check_defaults(self.config.get, '42')
+    def check_defaults(default: str) -> None:
+      assert self.config.get('c', 'fast') is None
+      assert self.config.get('c', 'preempt', None) is None
+      assert self.config.get('c', 'jake', default=default) == default
 
-  def test_default_section(self):
+    check_defaults('')
+    check_defaults('42')
+
+  def test_default_section(self) -> None:
     self.assertEqual('foo', self.config.get(Config.DEFAULT_SECTION, 'name'))
     self.assertEqual('foo', self.config.get(Config.DEFAULT_SECTION, 'name'))
 
-  def test_sections(self):
+  def test_sections(self) -> None:
     self.assertEqual(['a', 'b', 'defined_section'], self.config.sections())
 
-  def test_empty(self):
+  def test_empty(self) -> None:
     config = Config.load([])
     self.assertEqual([], config.sections())
-
-  def _check_defaults(self, accessor, default):
-    self.assertEqual(None, accessor('c', 'fast'))
-    self.assertEqual(None, accessor('c', 'preempt', None))
-    self.assertEqual(default, accessor('c', 'jake', default=default))
```

Comment:
This was inlined and `accessor` is no longer a parameter because it was always `self.config.get` for every call.